Changes

```diff
@@ -142,6 +142,10 @@ Working version
 
 ### Other libraries:
 
+- #9778: Fix printing for bindings where polymorphic type annotations and
+  attributes are present.
+  (Matthew Ryan)
```

Comment:
I would move the Changes entry to `compiler-libs` section ("Other libraries" refers to the contents of `otherlibs`).